vnext/DeforkingPatches/ReactCommon/yoga/yoga/Utils.cpp

```diff
@@ -0,0 +1,66 @@
+/*
```

Comment:
Why are these files here? Is it because we removed dependency on yoga lib somewhere?

Reply:
These are just modified files to used to override the original Yoga sources.

We still depend on Yoga same way as before.